Echo.Core/src/Textures/Colors/RGBA128.Parser.cs

```diff
@@ -171,56 +165,72 @@ bool ParseRGB(out RGBA128 result)
 			return YieldError(out result);

```

Comment:
You missed the comment about this code block being a duplicate of `ConvertRGBA` down below.

Reply:
I fixed that in the newest commit

Reply:
Oh lol did I not see that?